android/app/src/main/java/org/bitcoinppl/cove/sidebar/SidebarView.kt

```
@@ -63,15 +63,13 @@ fun SidebarView(
-            wallets =
-                try {
-                    Database().wallets().all()
-                } catch (e: Exception) {
-                    emptyList()
-                }
-        }
+    LaunchedEffect(app.isSidebarVisible, app.routeId) {
+        wallets =
+            try {
+                Database().wallets().all()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)